packages/astro/src/@types/astro.ts

```diff
@@ -1095,6 +1096,40 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		middleware?: boolean;
+		
+		/**
+		 * @docs
+		 * @name experimental.hybridOutput
+		 * Enable experimental support for opt-out prerendering
```

Comment:
Just checking that we're missing some fields here like `@type`, `@default`, `@version`, and `@description`?